### PR TITLE
Логанов А.С. 381806-2. ЛР3

### PR DESCRIPTION
**Pull request by mercy131**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

File: modules/intersection_line/src/intersect_app.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    bool tmp = l1.intersection(l2);
+    stream.str("");
+    if (tmp == true) {
+        stream << "Intersect!";
+    } else {
+        stream << "No Intersect!";
+    }
+    message_ = stream.str();
+    return message_;
+}
```
Comment (mercy131) [inline modules/intersection_line/src/intersect_app.cpp:80]: @Sergo567 , @drwff  проверьте пожалуйста.

File: modules/intersection_line/src/intersect_app.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 LoganovAndrei
+
+#include <stdlib.h>
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <utility>
+#include <cmath>
```
Comment (Sergo567) [inline modules/intersection_line/src/intersect_app.cpp:8]: cmath нигде не используется, лучше убери.

File: modules/intersection_line/CTestTests.txt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    COMMAND ${APPLICATION} 1 5 6
+)
+set_tests_properties (${prefix}_Check_Number_Of_Arguments PROPERTIES
+    PASS_REGULAR_EXPRESSION "Should be 9 arguments."
+    LABELS "${MODULE}")
+add_test(
+    NAME ${prefix}_Wrong_Number_Format
+    COMMAND ${APPLICATION} 1 2 3 qqq 5 6 7 8
+)
+set_tests_properties (${prefix}_Wrong_Number_Format PROPERTIES
```
Comment (drwff) [inline modules/intersection_line/CTestTests.txt:39]: Советую протестировать совпадающие отрезки

**Review by Sergo567 — APPROVED** [review]

**Review by drwff — APPROVED** [review]

**Review by OnlyDeniko — APPROVED** [review]
LGTM

**Pull request merged**